src/openfl/filters/BlurFilter.hx

```diff
@@ -245,16 +246,14 @@ private class BlurShader extends Shader {
 		
 		super ();
 		
-		#if !macro
-		data.uRadius.value = [ 0, 0 ];
-		#end
-		
 	}
 	
 	
 	private override function __update ():Void {
 		
-		data.uTextureSize.value = [ data.uImage0.input.width, data.uImage0.input.height ];
+		var u:ShaderParameterFloat2 = cast data.uTextureSize;
```

Comment:
Would be nice if there is a more explicit name of the parameter than just `u`. :)